src/PrimitiveObsession/IoC/EngineContainerBuilder.cs

```diff
@@ -1,16 +0,0 @@
-﻿using Autofac;
-
-namespace PrimitiveObsession.IoC
-{
-    public class EngineContainerBuilder
-    {
-        public IContainer Build(string tireCount, string pistonCount)
-        {
-            var builder = new ContainerBuilder();
-
-            builder.RegisterModule(new PrimitiveObsessionModule(pistonCount, tireCount));
-
-            return builder.Build();
-        }
-    }
-}
```

Comment:
Visto che ora abbiamo solo un modulo, sono d'accordo.

Ho adottato questa tecnica, perche' spesso si vanno a testare i singoli moduli di Autofac, ma non si testa mai veramente il punto di partenza dall'applicazione (lo Startup di Owin, etc) che spesso fanno delle registrazioni extra utili a testare i nostri moduli, e che quindi spesso si tende a copiare quello che fa nei punti di partenza anche nei test dei moduli.

Ripeto, in questo caso, tutto giusto, dobbiamo semplificare la lettura dell'articolo per il lettore, ergo, semplifichiamo.